css/src/tailwind.js

```diff
@@ -21,6 +21,19 @@ config.margin = Object.assign(config.margin, {
 
 config.negativeMargin = config.margin
 
+config.colors = Object.assign(config.colors, {
+  'black': '#333',
+  'grey-darkest': '#666',
+  'grey-darker': '#999',
+  'grey-dark': 'rgba(0,0,0,0.35)',
+  'grey': '#BFBFBF',
+  'grey-light': 'rgba(128,128,128,0.2)',
```

Comment:
What's the reasoning for using opacity to get the desired shade of grey? This is going to render differently depending on the parent's bg color

Reply:
This is just what Jeff has for his list of greys -- I anticipate this will change.